packages/api-explorer/src/JsonForm/configureJsonEditor.js

```diff
@@ -10,8 +10,59 @@ function setDefaultCustomization (JSONEditor) {
 
   editorsKeys.forEach(key => {
     JSONEditor.defaults.editors[key] = class Customization extends JSONEditor.defaults.editors[key] {
+      showEditJSON(){
+        super.showEditJSON()
+        if (this.editjson_holder && this.editjson_textarea && this.editjson_control) {
+          const outsideClickListener = event => {
+            if (!this.editjson_holder.contains(event.target) && this.editjson_holder.style.display !== 'none') {
+              this.hideEditJSON()
+              removeClickListener()
+            }
+          }
+          const removeClickListener = () => {
+            document.removeEventListener('click', outsideClickListener)
+          }
+          document.addEventListener('click', outsideClickListener)
+          this.editjson_textarea.style.width = '450px'
+          this.editjson_textarea.style.height = '340px'
+          this.editjson_holder.style.left = `${parseInt(this.editjson_holder.style.left, 10) -this.editjson_holder.offsetWidth - parseInt(this.editjson_control.style.marginLeft, 10)}px`
```

Comment:
If `this.editjson_holder.style.left` is undefined this line will generate NaN.

I'd calculate the size before, checking if it's safe and then set it

Reply:
Check added!